packages/components/upload/src/upload-list.vue

```diff
@@ -28,15 +24,15 @@
             file.status !== 'uploading' &&
             ['picture-card', 'picture'].includes(listType)
           "
-          class="el-upload-list__item-thumbnail"
+          :class="ns.be('list', 'item-thumbnail')"
           :src="file.url"
           alt=""
         />
-        <a class="el-upload-list__item-name" @click="handleClick(file)">
+        <a :class="ns.be('list', 'item-name')" @click="handleClick(file)">
           <el-icon class="el-icon--document"><document /></el-icon>
```

Comment:
You should also replace `el-list` and `el-icon`.

Please search regexp `["']el-` before committing.